src/pages/History/History.js

```diff
@@ -0,0 +1,24 @@
+import { Navbar, SideBar, LikedVideo } from "../../components";
+import { useLiked } from "../../context";
+
+export const History = () => {
+  const {
+    liked: { history }
+  } = useLiked();
+
+  return (
+    <>
+      <Navbar />
+      <div class="d-flex gap mg">
+        <SideBar />
+        <main class="main-video-container scrollable-element">
+          {history && history.length > 0 ? (
```

Comment:
You can use the optional chaining operator here

Reply:
ok will do that